cfy_manager/components/nginx/nginx.py

```diff
@@ -255,10 +255,8 @@ def install(self):
     def configure(self):
         logger.notice('Configuring NGINX...')
         self._configure()
-        if service._get_service_type() == 'supervisord':
-            service.configure(NGINX, append_prefix=False)
-            # Need to pass the override files
-        service.enable(NGINX, append_prefix=False)
+        if service._get_service_type() != 'supervisord':
+            service.enable(NGINX, append_prefix=False)
```

Comment:
How about making the logic for this be inside the service so that supervisord knows to just ignore it?
Alternatively, maybe it should be part of the start command, where systemd would enable+start on cfy_manager start and stop+disable on cfy_manager stop?

Reply:
Well, this case is only valid for nginx as we need to enable other services, as nginx configurations contains some references to internal/external certificate and running `supervisorctl -c /etc/supervisord.conf update nginx` will raise error that these files are not exist yet and that the reason why service.configure() exists on the start action`. Moreover, `service.configure(NGIXN)` is only valid for supervisord and not for systemd as the service unit file will be created once we install the nginx. 

```

Reply:
OK, marking as resolved.